Firestore/Source/API/FIRFirestore+Internal.h

```diff
@@ -65,6 +65,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)shutdownWithCompletion:(nullable void (^)(NSError *_Nullable error))completion
     NS_SWIFT_NAME(shutdown(completion:));
 
+- (void)clearPersistenceWithCompletion:(nullable void (^)(NSError *_Nullable error))completion;
```

Comment:
This will eventually be in the public header, so this should get the full comment treatment (i.e. taking what's in the API review's comment and adapting it to Objective-C).

Reply:
The JavaScript version of this is much better. We should adapt that to Objective-C/Swift and use that text.

Reply:
Make sure to incorporate our final round of edits (generally this means using the Swift names for things).